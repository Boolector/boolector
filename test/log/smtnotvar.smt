(benchmark smttrue.smt
 :logic QF_BV
 :status sat
 :extrapreds ((a))
 :formula (not a))
