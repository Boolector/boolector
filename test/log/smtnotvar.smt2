(set-option :incremental false)
(set-info :status sat)
(set-logic QF_BV)
(declare-fun a () Bool)
(assert (not a))
(check-sat)

