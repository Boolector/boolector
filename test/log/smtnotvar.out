sat
2 0 a
